manual/parts/threads.tex

```diff
@@ -658,11 +658,28 @@ \section{Virtualisation}
 
 \subsection{Arm}
 
-When a \obj{TCB} has a bound \obj{VCPU} it is allowed to have the mode portion of the
-\texttt{cpsr} register set to values other than \texttt{user}. Specifically it may have any value other than
-\texttt{hypervisor}.
+When a \obj{TCB} has a bound \obj{VCPU} it will have access to (virtualised) system registers,
+cache and TLB maintenance instructions and be able to handle some exceptions itself.
+The virtual GIC will be enabled, allowing virtual interrupt delivery.
 
-TODO: this section needs more detail
+The vitualised system registers can be modified with \apifunc{seL4\_ARM\_VCPU\_WriteRegs}{arm_vcpu_write_registers}.
```

Comment:
typo: vitualised -> virtualised

Reply:
Thanks, but it's a bit too early for review, this is a placeholder for future commits. I currently have a mess of local modifications I'm still in the process of untangling, but didn't got to yet. I plan to do that the coming weeks.

That typo was part of a work-in-progress commit that shouldn't be part of this PR, would you like to review #1232 which covers ARM virtualisation?

Reply:
Ok, I can do that (not being an expert for ARM).